lib/RoadizCoreBundle/src/Console/UsersCreationCommand.php

```diff
@@ -62,6 +62,14 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         ];
         $passwordInput = new ArrayInput($arguments);
 
+        if ($plainPassword = $input->getOption('plain-password')) {
```

Comment:
You should test plain-password length here. Because if it's too short your sub-command will fail, but user will still be created.